website/index.html

```diff
@@ -1,37 +1,84 @@
+  
   <link rel="shortcut icon" href="https://cloud.google.com/images/gcp-favicon.ico">
-  <link rel="stylesheet" href="src/vendor-d362a4bbd8.css">
-  <link rel="stylesheet" href="src/app-a103c713b2.css">
-</head>
+  <link rel="canonical" href="//googlecloudplatform.github.io/google-cloud-powershell/">
+
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Open+Sans:100,300,400,600,700,800" />
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto:100,300,400,500" />
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Roboto+Mono" />
+  <link rel="stylesheet" type="text/css" href="/google-cloud-powershell/css/style.css" />
 
-<body>
-  <!--[if lt IE 10]>
- <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
- <![endif]-->
-  <main ui-view="" autoscroll="true"></main><script src="src/vendor-d6198d67f2.js"></script><script src="src/app-780bf1f09e.js"></script><script>
-      (function() {
-        var $injector = angular.injector(['ng']);
-        var $http = $injector.get('$http');
+  <base href="/google-cloud-powershell/">
+
+  <script src="/google-cloud-powershell/js/angular.min.js"></script>
+  <script src="/google-cloud-powershell/js/angular-route.min.js"></script>
+  <script src="/google-cloud-powershell/js/angular-sanitize.min.js"></script>
+
+  <script src="/google-cloud-powershell/js/app/app.js"></script>
+  <script src="/google-cloud-powershell/js/app/controllers/content.js"></script>
+  <script src="/google-cloud-powershell/js/app/controllers/cmdlet-explorer.js"></script>
+  <script src="/google-cloud-powershell/js/app/controllers/parameters-widget.js"></script>
+  <script src="/google-cloud-powershell/js/app/controllers/syntax-widget.js"></script>
+</head>
+<body ng-app="powershellSite">
+  <div id="header">
+    <div class="psr-headerIcon"></div>
+    <a href="/google-cloud-powershell/#/">
+      <p class="psr-headerText">Cloud Tools for PowerShell</p>
+    </a>
+  </div>
 
-        $http.get('manifest.json')
-          .then(function(response) {
-            angular
-              .module('gcloud.manifest', [])
-              .constant('manifest', response.data);
+  <div class="psr-main" ng-controller="ContentController as contentCtrl">
+    <div id="left-nav">
+      <cmdlet-explorer documentation-obj="contentCtrl.cmdletDocumentation.products"></cmdlet-explorer>
+    </div>
+    <div id="content-container">
 
-            angular.element(document)
-              .ready(function() {
-                angular.bootstrap(document, ['gcloud']);
-              });
-          });
-      }());
-    </script></body>
+      <div id="top-nav-container">
+        <div id="top-nav">
+          <div class="top-navSection psr-navText">
+            <a href="/google-cloud-powershell/#/">
```

Comment:
I definitely agree with the point, but I'm not convinced it would be net simpler. e.g. using a filter:

` <a href="{{ '/' | makeLink }}">`

or use a value on $scope

` <a href="{{ linkPrefix }}/">`

Given that it would be one-line to bulk fix using `grep` and `sed` I'm inclined to leave it as-is.